static/scripts/jquery.validation.js

```diff
@@ -0,0 +1,54 @@
+(function( $ ) {
+ 
+    $.fn.wppStatelessValidate = function(options, response) {
+
+        var _this = jQuery(this);
+
+        if (typeof _this.data('settings') == 'undefined'){
+            var _settings = $.extend({
+                name: {},
+                id: {},
+            }, options );
+
+            _this.data('settings', _settings);
+        }
+
+        var pName = _this.val();
+        var settings = _settings || _this.data('settings');
+
+        response = response || {};
```

Comment:
@alimuzzaman when working with objects and setting properties I really recommend you use `lodash`. Its a great tool. You can do stuff like:

```js
_.set( response, 'some.long.path', 'value here' );
```
Its more future proof and easier to work with, but just an opinion.